.erb-lint.yml

```diff
@@ -1,8 +1,20 @@
 ---
+EnableDefaultLinters: true
 linters:
   ErbSafety:
     enabled: true
     exclude:
       - '**/app/views/layouts/_google_tag_manager.html.erb'
   HardCodedString:
     enabled: false
+  Rubocop:
+    enabled: true
+    rubocop_config:
+      inherit_from:
+        - .rubocop.yml
+      Layout/InitialIndentation:
+        Enabled: false
+      Layout/TrailingEmptyLines:
+        Enabled: false
+      Lint/UselessAssignment:
+        Enabled: false
```

Comment:
`erblint` evaluates Ruby code in isolation, so these rules don't make any sense.